Makefile

```diff
@@ -317,6 +317,10 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+
+compose-kc:
```

Comment:
this is not needed as `Makefile` contains a method for calling individual modules from `lib/`